### DISCUSSION TITLE
Barb just rerolled enigma :D

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#317] [General]
Barb just rerolled (hel+tp) enigma Breast Plate @ level 63, i had put it in his stash by hand ( eni is 65 req not sure if that's important). Also my Assa didn't equip the eni by herself ( same bp ), had to put it on by hand then she started respec.

**Comment by isid0re** [comment]
Bot is not looking for enigma in a breastplate. The lowest str breastplate is mageplate. If you wanted to have it use a breastplate you will need to have it added to a wanted armor for enigma and add the breastplate option to the making of enigma and keeprunewords for enigma as well.
